### ISSUE TITLE
[API Usability] MxInt4 x Bf16 TRT-LLM Gen MoE

### ISSUE THREAD

**Issue by HandH1998** [flashinfer-ai/flashinfer#2230]
Hello! I noticed Flashinfer has supported MxInt4 x Bf16 TRT-LLM Gen MoE in https://github.com/flashinfer-ai/flashinfer/pull/2159. But I found only tests for `RoutingMethodType.Renormalize` are added in [test_renormalize_routing](https://github.com/flashinfer-ai/flashinfer/blob/main/tests/moe/test_trtllm_gen_fused_moe.py#L2595). I tried adding a similar test case for `RoutingMethodType.DeepSeekV3` in [test_deepseekv3_routing](https://github.com/flashinfer-ai/flashinfer/blob/f0355f76922a762222b098ef434fcfedff367dd5/tests/moe/test_trtllm_gen_fused_moe.py#L2595), but it failed with an error.

<img width="2488" height="644" alt="Image" src="https://github.com/user-attachments/assets/5ebf90a1-b657-4932-9b7a-5354289d919b" />

Could you confirm if trtllm_mxint4_block_scale_moe currently supports `RoutingMethodType.DeepSeekV3`? Thanks!

Reproduce:
run https://gist.github.com/HandH1998/b74032e087b30b2a6148b4c9966f8020 
```
pytest test_trtllm_gen_fused_moe.py::test_deepseekv3_routing
```

**Comment by ispobock** [comment]
@nekorobov @yzh119 Could you take a look?

**Comment by nekorobov** [comment]
Hi, from a quick look the error happens in the routing kernel. It could be that the error is simply a wrong type of the routing bias or the routing logits. I will have a look.

**Comment by HandH1998** [comment]
> Hi, from a quick look the error happens in the routing kernel. It could be that the error is simply a wrong type of the routing bias or the routing logits. I will have a look.

Thanks for your feedback. This issue may be related to the following code: at https://github.com/flashinfer-ai/flashinfer/blob/main/flashinfer/fused_moe/core.py#L2643, the `trtllm_mxint4_block_scale_moe` receives `None` as the default value for `routing_bias`.

**Comment by nekorobov** [comment]
I think, you are right. Thank you for the analysis! Bias is optional down the stack and it is never checked that bias is required for DeepSeek routing.

**Comment by nekorobov** [comment]
Adding support in https://github.com/flashinfer-ai/flashinfer/pull/2234. @HandH1998 , please verify and thank you again for reporting!

**Comment by HandH1998** [comment]
@nekorobov Thank you for your fixing. Now the API can work without throwing error. But `trtllm_bf16_moe` and `trtllm_mxint4_block_scale_moe` still have incorrect moe outputs for `RoutingMethodType.DeepSeekV3`. Please check if the two APIs have supported `RoutingMethodType.DeepSeekV3`. Thanks!

**Comment by aleozlx** [comment]
tagged the issue to keep it on my radar

**Comment by HandH1998** [comment]
> @nekorobov Thank you for your fixing. Now the API can work without throwing error. But `trtllm_bf16_moe` and `trtllm_mxint4_block_scale_moe` still have incorrect moe outputs for `RoutingMethodType.DeepSeekV3`. Please check if the two APIs have supported `RoutingMethodType.DeepSeekV3`. Thanks!

I have comfirmed it can work for deepseekv3. Thanks！ This issue can be closed.
